main.js

```diff
@@ -0,0 +1,48 @@
+import { app, BrowserWindow } from "electron"
```

Comment:
Also note we prefer *not* to host project's code on Flathub, just packaging files: https://github.com/flathub/flathub/wiki/App-Maintenance#strongly-discouraged